docs/pipeline-components-and-applications/enrichment-components/configuration-reference/index.md

```diff
@@ -264,6 +220,8 @@ If `acceptInvalid` is set to `false`, a bad row will be emitted instead of the e
 
 When we'll know that all our customers don't have any invalid enriched events any more, we'll remove the feature flags and it will be impossible to emit invalid enriched events.
 
+Since `4.0.0`, it is possible to configure the lengths of the atomic fields with `validation.atomicFieldsLimits` (see above).
```

Comment:
Worth copy-pasting the example here I think — the one from the “upgrading” page.